### DISCUSSION TITLE
ESP32 Display SSD1306 vertical bar from top to bottom displayed

### DISCUSSION THREAD

**Discussion by italschwob** [arendst/Tasmota#10764] [Q&A] [answered]
Hi, 
have a problem with the DISPLAY SSD1306, when I enter DisplayModel 2 in the console, the response is DisplayModel 7.
I2CScan = 0x3c
I have a ESP32 Tasmota 9.2.0.2 compiled by myself, SSD1306 128x64 with the I2C address 0x3c, There is a vertical bar from top to bottom displayed, texts are all displayed correctly, just there is always this bar. Have I forgotten something, what is still to be set here?
![IMG_20210130_201936](https://user-images.githubusercontent.com/46403511/106365987-58db8800-6339-11eb-95b2-89004aad0568.png)
I have already replaced the display, the problem remains

**Comment by ascillato** [comment, marked as answer]
Please, address this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4). The chat is a better and more dynamic channel for helping you. 

Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
